README.md

```diff
@@ -14,92 +13,50 @@
+## What is Octobot  ?
 
-To install OctoBot with its tentacles, just use the [latest release for your system](https://github.com/Drakkar-Software/OctoBot/releases/latest) and your OctoBot is ready ! 
+<a href="https://www.octobot.online/guides/#telegram"><img src="../assets/telegram-interface.png" height="414" alt="Telegram interface"></a>  
+<a href="../assets/dashboard"><img src="../assets/dashboard" height="414" alt="Web interface"></a>  
 
-Find the answers to the most common questions in [our FAQ](https://www.octobot.info/usage/frequently-asked-questions-faq).
+[Octobot](https://www.octobot.online/) is a powerful open-source cryptocurrency trading robot.
 
-## Your Octobot
-<a href="https://www.octobot.online/guides/#telegram"><img src="../assets/telegram-interface.png" height="414" alt="Telegram interface"></a>
-[![Twitter Interface](../assets/twitter-interface.png)](https://www.octobot.info/interfaces/twitter-interface)
+OctoBot is highly customizable using its configuration and tentacles system.  
+You can build your own bot using the infinite [configuration](https://www.octobot.online/guides/#trading_modes) possibilities such as  
+**technical analysis**, **social media processing** or even **external statistics management** like google trends.  
+  
+OctoBot is **AI ready**: Python being the main language for OctoBot, it's easy to integrate machine-learning libraries such as [Tensorflow](https://github.com/tensorflow/tensorflow) or  
+any other libraries and take advantage of all the available data and create a very powerful trading strategy.  
+  
+Octobot's main feature is **evolution**, you can : 
+- share your configurations with other octobot users 
+- [install](https://www.octobot.info/advanced_usage/tentacle-manager), [modify](https://developer.octobot.info/tentacles/tentacle-development) and even [create](https://developer.octobot.info/tentacles/tentacle-development) new tentacles to build your ideal cryptocurrency trading robot
+- [contribute](https://developer.octobot.info/installation/developer-installation/octobot-developer-installation) to improve OctoBot core repositories and tentacles
 
-OctoBot is highly customizable using its configuration and tentacles system. 
-You can build your own bot using the infinite [configuration](https://www.octobot.online/guides/#trading_modes) possibilities such as 
-**technical analysis**, **social media processing** or even **external statistics management** like google trends.
+## Installation  
+OctoBot's installation is **very simple**, you can either:
+- [Deploy your OctoBot on OctoBot Cloud](https://octobot.cloud/). On OctoBot Cloud, your robot will be running and reachable at all time, it will also benefit from cloud only features.
+- [Download and install](https://www.octobot.info/installation/local-installation) OctoBot on your own computer or server. This enables you to use OctoBot for free.
+- Install OctoBot [using docker](https://www.octobot.online/local_installation/).
```

Comment:
```suggestion
- Install OctoBot [using docker](https://www.octobot.online/with-docker).
```

Reply:
right, i changed it to https://www.octobot.info/installation/local-installation#option-2-with-docker